vscode/src/non-stop/FixupController.ts

```diff
@@ -996,9 +996,30 @@ export class FixupController
 
         // append response to new file
         const doc = await vscode.workspace.openTextDocument(newFileUri)
-        const pos = new vscode.Position(Math.max(doc.lineCount - 1, 0), 0)
-        const range = new vscode.Range(pos, pos)
+        const lastLine = doc.lineAt(Math.max(doc.lineCount - 1, 0))
+
+        const EOF = lastLine.rangeIncludingLineBreak.end
+
+        const range = new vscode.Range(EOF, EOF)
```

Comment:
```suggestion
        task.selectionRange = new vscode.Range(EOF, EOF)
```

If we are resetting task.selectionRange below, we won't need `range` anymore?